instances/atlas/parcellationEntityVersion/Brodmann_1909/BA_1909_cerebralCortex.jsonld

```diff
@@ -0,0 +1,14 @@
+
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/BA_1909_cerebralCortex",
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": null,
+    "hasVersion": null,
+    "lookupLabel": "BA_1909_cerebralCortex",
+    "name": "Brodmann cerebral cortex",
+    "ontologyIdentifier": null,
+    "relatedUBERONTerm": "http://purl.obolibrary.org/obo/UBERON_0000956"
```

Comment:
This is wrong. It needs to point to a controlled term from the UBERON terminology. In this case: "https://openminds.ebrains.eu/instances/UBERONParcellation/cerebralCortex".

Reply:
this is correct. and also wrong in other cases. I'll fix this one in this PR and then run another one for correcting the rest.